etna/models/base.py

```diff
@@ -78,7 +78,7 @@ def forecast(
 
     @staticmethod
     def _forecast_segment(model, segment: Union[str, List[str]], ts: TSDataset) -> pd.DataFrame:
-        segment_features = ts[:, segment, :]
```

Comment:
Hm, is it ok, that we make this in the task about tests?

Reply:
It influences the results of these tests, so I decided that it will be faster to make it here than making in another task and waiting for merge.

Reply:
🆗
 I pray we've not broken something.

Reply:
What difference to broking smth will it make if I do it in other task?)

Reply:
It's easier to make reverts for example if we not make multiple task at once.

Reply:
Ok, I'll make it in other PR.